BnbnavNetClient/Views/MapView.axaml.cs

```diff
@@ -453,6 +487,79 @@ public override void Render(DrawingContext context)
+            if (_svgCache.TryGetValue(uriString, out var outSvg))
+            {
+                svg = outSvg;
+            }
+            else
+            {
+                var uri = new Uri(uriString);
+                if (_assetLoader.Exists(uri))
+                {
+                    var asset = _assetLoader.Open(uri);
+
+                    svg = new();
+                    svg.Load(asset);
+                    if (svg.Picture is null)
+                        return;
+                    _svgCache.Add(uriString, svg);
+                }
+            }
+
+            if (svg is null)
+                return;
+
+            const int playerSize = 48;
+            var rect = new Rect(ToScreen(player.MarkerCoordinates) - new Point(playerSize, playerSize) / 2, new Size(playerSize, playerSize));
+
+            var sourceSize = new Size(svg.Picture!.CullRect.Width, svg.Picture.CullRect.Height);
+            var scaleMatrix = Matrix.CreateScale(
+                rect.Width / sourceSize.Width,
+                rect.Height / sourceSize.Height);
+            var translateMatrix = Matrix.CreateTranslation(
+                rect.X * sourceSize.Width / rect.Width,
+                rect.Y * sourceSize.Height / rect.Height);
+            var rotateMatrix = Matrix.CreateRotation(-player.MarkerAngle * Math.Tau / 360.0);
+            var preRotateMatrix = Matrix.CreateTranslation(-sourceSize.Width / 2, -sourceSize.Width / 2);
+
+            // context.DrawEllipse(new SolidColorBrush(player.SnappedEdge is null ? new Color(255, 255, 0, 0) : new Color(255, 0, 255, 0)), null, ToScreen(player.MarkerCoordinates), 50, 50);
+            // context.DrawLine(new Pen(new SolidColorBrush(new Color(255, 0, 255, 0))), ToScreen(player.Velocity.Point1), ToScreen(player.Velocity.Point2));
+            
+            using (context.PushClip(rect))
+            using (context.PushPreTransform(translateMatrix * scaleMatrix))
+            using (context.PushTransformContainer())
+            using (context.PushPostTransform(preRotateMatrix))
+            using (context.PushPostTransform(rotateMatrix))
+            using (context.PushPostTransform(preRotateMatrix.Invert()))
+                context.Custom(new SvgCustomDrawOperation(rect, svg));
+            
+            //Draw the player name
+            var textBrush = (Brush)this.FindResource("ForegroundBrush")!;
+            player.PlayerText.SetForegroundBrush(textBrush);
+
+            var textCenter = rect.Center + new Point(0, rect.Height / 2 + 10 + player.PlayerText.Height / 2);
+            context.DrawText(player.PlayerText, textCenter - new Point(player.PlayerText.Width, player.PlayerText.Height) / 2);
+
+            if (player.SnappedEdge is not null)
+            {
+                var roadText = new FormattedText(player.SnappedEdge.Road.Name, CultureInfo.CurrentCulture, FlowDirection.LeftToRight,
+                    new("Noto Sans"), 20, new SolidColorBrush(new Color(255, 255, 255, 255)));
```

Comment:
check whether this is actually grabbing the noto sans from the resources or the noto sans from the system